rdkit/ML/AnalyzeComposite.py

```diff
@@ -83,7 +83,7 @@ def ProcessIt(composites, nToConsider=3, verbose=0):
     if verbose >= 0:
       print('# Average Descriptor Positions')
     retVal = []
-    for k in globalRes.keys():
+    for k in sorted(globalRes.keys()):
```

Comment:
good point about putting the sorted in the test. I'll fix that and the second comment.